packages/react-chat/src/components/UserResponse/styled.ts

```diff
@@ -20,13 +20,27 @@ export const Container = styled(tag('div'), {
   flexShrink: 0,
   alignItems: 'flex-end',
 
-  [`& ${Message.Container}`]: {
-    maxWidth: 282,
-  },
-
   [`& > ${Tooltip.Container}`]: {
     marginTop: '$1',
   },
+  variants: {
+    mode: {
```

Comment:
we could maybe treat `bubble` and `embedded` as themes so that we don't need to props drill for the styled components

Reply:
made a ticket for this https://linear.app/voiceflow/issue/DX-1025/refactor-chat-modes-into-stitches-themes